.pre-commit-config.yaml

```diff
@@ -5,15 +5,17 @@ repos:
       - id: check-merge-conflict
       - id: debug-statements
       - id: trailing-whitespace
+        exclude: ^main/static/assets/
       - id: end-of-file-fixer
       - id: pretty-format-json
+        exclude: ^main/static/assets/
         args: [--autofix, --indent, '4', --no-sort]
   - repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
     rev: v2.14.0
     hooks:
       - id: pretty-format-yaml
```

Comment:
This is currently formatting yaml files. Is it likely to class with prettier? Also, would it be worth adding some of these new linters to the pre-commit config (perhaps in a separate PR)?